dongkyu/initial-settings-express/myapp/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
+
+DATABASE_URL = URL
```

- 127.0.0.1은 로컬을 가리키는 host 정보입니다.
개발 환경에 따라 127.0.0.1이 아닐 수도 있기 때문에 hostNumber 혹은 host와 같은 보편적인 단어로 수정해주는 것이 좋을 것 같습니다!
- 3306은 MySQL 데이터베이스를 가리키는 고유 port 번호입니다. 다른 RDBMS를 사용할 수도 있으며, 해당 RDBMS를 유추할 수 있는 정보는 제외하는 것이 더 좋을 것 같습니다!
- 3번째 코드 라인은 중복이 되고 있으며 불필요한 코드인 것 같으니 삭제해주세요!

```suggestion
DATABASE_URL="mysql://username:password@host:port/database-name"
```